src/test/java/com/jcabi/dynamo/mock/H2DataTest.java

```diff
@@ -134,7 +134,28 @@ public void createsTablesWithLongNames() throws Exception {
      */
     @Test
     public void supportsTableNamesWithIllegalCharacters() throws Exception {
-        new H2Data().with(".-.", new String[] {"pk"}, new String[0]);
+        new H2Data().with(".-.", new String[]{"pk"}, new String[0]);
+    }
+
+    /**
+     * H2Data supports column names with characters illegal to H2.
+     * @throws Exception In case test fails
+     */
+    @Test
+    @Ignore
+    public void supportsColumnNamesWithIllegalCharacters() throws Exception {
+        /* todo: #28:30min H2Data doesn't support COLUMNS with
```

Comment:
can't out <code>todo</code> under <code>@throws</code>
it provokes check style error:
<pre>
[ERROR] /src/test/java/com/jcabi/dynamo/mock/H2DataTest.java[143]: Should contain one indentation space (MultilineJavadocTagsCheck)
</pre>